qiskit/transpiler/passes/layout/vf2_post_layout.py

```diff
@@ -143,249 +129,40 @@ def run(self, dag):
         """run the layout method"""
         if self.target is None:
             raise TranspilerError("A target must be specified or a coupling map must be provided")
-        if not self.strict_direction:
-            self.avg_error_map = self.property_set["vf2_avg_error_map"]
-            if self.avg_error_map is None:
-                self.avg_error_map = vf2_utils.build_average_error_map(self.target, None)
```

Comment:
This comment is intended for the lines above these, where it says:
```python
if self.target is None:
        raise TranspilerError("A target must be specified or a coupling map must be provided")
```
This pass does not accept the `coupling_map` argument! 😄 

Modulo that, the pass has the same arguments as `VF2Layout`, but somehow in a different order.

Reply:
yeah, don't blame me for that one lol.  Fixed the exception message in df6d109671.